packages/clerk-js/src/core/resources/SignUp.ts

```diff
@@ -81,54 +81,25 @@ export class SignUp extends BaseResource implements SignUpResource {
     this.fromJSON(data);
   }
 
-  create = async (params: SignUpCreateParams): Promise<SignUpResource> => {
-    const paramsWithCaptcha: Record<string, unknown> = params;
-
-    if (!__BUILD_DISABLE_RHC__) {
-      const {
-        captchaSiteKey,
-        canUseCaptcha,
-        captchaURL,
-        captchaWidgetType,
-        captchaProvider,
-        captchaPublicKeyInvisible,
-      } = retrieveCaptchaInfo(SignUp.clerk);
-
-      if (
-        !this.shouldBypassCaptchaForAttempt(params) &&
-        canUseCaptcha &&
-        captchaSiteKey &&
-        captchaURL &&
-        captchaPublicKeyInvisible
-      ) {
-        try {
-          const captchaParams = await getCaptchaToken({
-            siteKey: captchaSiteKey,
-            widgetType: captchaWidgetType,
-            invisibleSiteKey: captchaPublicKeyInvisible,
-            scriptUrl: captchaURL,
-            captchaProvider,
-          });
-
-          paramsWithCaptcha.captchaToken = captchaParams.captchaToken;
-          paramsWithCaptcha.captchaWidgetType = captchaParams.captchaWidgetType;
-        } catch (e) {
-          if (e.captchaError) {
-            paramsWithCaptcha.captchaError = e.captchaError;
-          } else {
-            throw new ClerkRuntimeError(e.message, { code: 'captcha_unavailable' });
-          }
-        }
+  create = async (_params: SignUpCreateParams): Promise<SignUpResource> => {
+    let params: Record<string, unknown> = _params;
+
+    if (!__BUILD_DISABLE_RHC__ && !this.clientBypass() && !this.shouldBypassCaptchaForAttempt(params)) {
+      const captchaChallenge = new CaptchaChallenge(SignUp.clerk);
+      const captchaParams = await captchaChallenge.managed();
```

Comment:
Do we force managed here? Because it could be invisible, right?

Reply:
`managed` will default to the settings used in env, `invisible` forces invisible

Reply:
The naming is a bit confusing here maybe. The managed somehow implies it uses the managed turnstile challenge.

Reply:
Correct, but this is the flow for instances that have selected the invisible setting also. I suggest we rename this function to something like `runChallenge()` or something that includes the invisible as a valid option and not only as a fallback.